openmetadata-spec/src/main/resources/json/schema/type/entityLineage.json

```diff
@@ -124,6 +128,10 @@
         "$ref": "#/definitions/edge"
       },
       "default": null
+    },
+    "description" :{
+      "description": "description of lineage",
+      "type": "string"
```

Comment:
This description field I believe is not required as we are already getting the description in the lineageDetails object, so this looks like an ambiguous field

Reply:
done